dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -831,3 +831,7 @@ api.system.ruleengine.actionlet.VisitorTagsActionlet.MAX_TAGS=20
 
 ## Translation actionlet service key
 #GOOGLE_TRANSLATE_SERVICE_API_KEY=put-your-key-here
+
+## Number of site to display on the dot-site-selector component per filtering request
+DOTSITESELECTOR_SITES_PER_PAGE=20
```

Comment:
I think this should be optional in the dotmarketing-config. It isn't @jgambarios